ui/src/components/navigation/navigation-bar.module.scss

```
@@ -59,15 +59,15 @@
 
 .itemCount {
   display: block;
-  @include body-xlarge();
+  @include body-xlarge;
   font-weight: 600;
```

Stylelint (17.12.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)